modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,124 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage_{context}""]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+To configure your heterogeneous cluster, you must create an `arm64` bootimage and add it to your Azure machine set. The following procedure describes how to manually generate an `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+$ az login
+----
+. Create a storage account and upload the `arm64` virtual hard disk (VHD) to your storage account. The {product-title} installation program creates a resource group, however, the bootimage can also be uploaded to a custom named resource group: 
++
+[source,terminal]
+----
+$ az storage account create -n ${STORAGE_ACCOUNT_NAME} -g ${RESOURCE_GROUP} -l westus --sku Standard_LRS <1>
```

Comment:
I looked into the formating for the names and the `${}` is correct for using the `az` command. 
That was my mistake, we usually use `<>` in other parts of the docs.